dcos/dcos-ui
# DCOS-55638: ServiceTreeView - No limit banner & badges

Closes https://jira.mesosphere.com/browse/DCOS-55638 .

## Testing
0. Start localhost and login.
1. Update your `Config.dev.ts`
```
uiConfiguration: {
...
  features: {
    quota: true
  }
},
  useFixtures: true,
  ...
```
2. On services page, click "2_apps" group.
3. Verify that the infobox and the badge are shown.
4. Verify that the infobox and the badge are not shown for other groups.
5. In `tests/_fixtures/marathon-pods/groups.js` change `enforceRole` on line 281 to false and verify that he badge and the infobox are no longer shown.
6. Verify that the added test makes sense.

## Trade-offs
None.

## Dependencies
None.

## Screenshots
![Снимка от 2019-07-17 11-25-56](https://user-images.githubusercontent.com/40791275/61359755-aa224480-a885-11e9-99a5-9f46c434294f.png)
